lib/create_linkedin_oauth_config.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+import type { OAuth2ClientConfig } from "../deps.ts";
+import { getRequiredEnv } from "./get_required_env.ts";
+
+/**
+ * Returns the OAuth configuration for LinkedIn.
+ *
+ * Requires `--allow-env[=LINKEDIN_CLIENT_ID,LINKEDIN_CLIENT_SECRET]` permissions
+ * and environment variables:
+ * 1. `LINKEDIN_CLIENT_ID`
+ * 2. `LINKEDIN_CLIENT_SECRET`
+ *
+ * @example
+ * ```ts
+ * import { createLinkedInOAuthConfig } from "https://deno.land/x/deno_kv_oauth@$VERSION/mod.ts";
+ *
+ * const oauthConfig = createLinkedInOAuthConfig({
+ *  redirectUri: "http://localhost:8000/callback",
+ *  scope: "r_liteprofile r_emailaddress"
```

Comment:
These scopes look deprecated now. https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin

New scopes are, for example, `profile email`